src/main/java/com/openshift/internal/restclient/model/ReplicationController.java

```diff
@@ -364,4 +366,24 @@ public void addVolume(IVolumeSource volumeSource) {
 		}
 		volList.add(ModelNode.fromJSONString(volumeSource.toJSONString()));
 	}
+
+    @Override
+    public void setServiceAccount(String serviceAccount) {
```

Comment:
Are you saying we need to add it into the IPod interface/Impl as well? I was kind of wondering why the RC does not implement the pod interface.